bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/eco-channel-types.xml

```diff
@@ -4,25 +4,25 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 	<channel-type id="accel">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="%">Number:Dimensionless</item-type>
 		<label>Acceleration Score</label>
 		<description>Score for smooth acceleration</description>
 		<state readOnly="true"/>
 	</channel-type>
 	<channel-type id="coasting">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="%">Number:Dimensionless</item-type>
```

Comment:
Just a double check: the scores are percentages and not just a number?

Reply:
Yes, it's percentage